.github/workflows/dockerhub.yml

```diff
@@ -89,12 +93,34 @@ jobs:
       - name: Determine Build Metadata
         id: build-metadata
         uses: usdot-fhwa-stol/actions/docker-build-metadata@main
+        
+      - name: Process Suffix in Output Docker Image tag (handle remove_suffix)
```

Comment:
Just want to know for remove_suffix if a branch name is something like feature-develop-humble or current branch arc-248-remove-suffix-dockerhub, is this going to remove everything after the first dash and losing rest after that ?

Reply:
good catch. Yeah we should only remove past the last dash. Let me revise

Reply:
Actually I realized I could test that locally without wasting time setting up the github action because its just bash action. So I confirmed it does remove only last

Reply:
```
            # ${variable%-pattern} removes the shortest match of pattern from the end
            # Example: "develop-humble" becomes "develop"
            BASE_TAG="${ORIGINAL_TAG%-*}"
            
            # ${variable##pattern} removes the longest match of pattern from the beginning
            # Example: for "develop-humble" this extracts just "humble"
            REMOVED_SUFFIX="${ORIGINAL_TAG##*-}"
```